src/entities/CheckScenes/task/taskRunnerSqs.ts

```diff
@@ -53,17 +53,16 @@ export async function taskRunnerSqs(job: DeploymentToSqs) {
 
   let placesToProcess: ProcessEntitySceneResult | null = null
 
-  if (
-    contentEntityScene.metadata.worldConfiguration &&
-    !(
-      contentEntityScene.metadata.worldConfiguration.name ||
-      contentEntityScene.metadata.worldConfiguration.dclName
-    )
-  ) {
-    throw new Error("worldConfiguration without name")
-  }
-
   if (contentEntityScene.metadata.worldConfiguration) {
+    if (
+      !(
+        contentEntityScene.metadata.worldConfiguration.name ||
+        contentEntityScene.metadata.worldConfiguration.dclName
+      )
+    ) {
+      throw new Error("worldConfiguration without name")
+    }
```

Comment:
What do you think about doing this as a separate if, like 

```suggestion
    if (
       contentEntityScene.metadata.worldConfiguration && 
      !(
        contentEntityScene.metadata.worldConfiguration.name ||
        contentEntityScene.metadata.worldConfiguration.dclName
      )
    ) {
      throw new Error("worldConfiguration without name")
    }
```